reporter/urlparams.js

```diff
@@ -85,7 +86,7 @@ function getUrlParams() {
 			// Allow just a key to turn on a flag, e.g., test.html?noglobals
 			value = param.length === 1 ||
 				decodeQueryParam( param.slice( 1 ).join( "=" ) ) ;
-			if ( urlParams[ name ] ) {
+			if ( hasOwn.call( urlParams, name ) ) {
```

Comment:
HA! Now I just saw this comment...

Well, the Object.create(null) is great, but we can stick with the hasOwn :)